src/signalling/room.rs

```diff
@@ -242,7 +248,7 @@ impl Room {
         let to_peer_id = from_peer.partner_peer_id();
         let to_peer = self.peers.get_peer(to_peer_id)?;
         if let PeerStateMachine::New(_) = to_peer {
-            return Err(RoomError::PeerStateError(PeerStateError::WrongState(
+            return Err(RoomError::PeerError(PeerError::WrongState(
```

Comment:
@tyranron ,

Так линтер ругается:

```
error: returning an `Err(_)` with the `?` operator
   --> src/signalling/room.rs:250:13
    |
250 | /             Err(PeerError::WrongState(
251 | |                 to_peer_id,
252 | |                 "Not New",
253 | |                 format!("{}", to_peer),
254 | |             ))?
    | |_______________^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#try_err
help: try this
    |
250 |             return Err(PeerError::WrongState(
251 |                 to_peer_id,
252 |                 "Not New",
253 |                 format!("{}", to_peer),
254 |             ).into())
```

Причина:

> The ? operator is designed to allow calls that can fail to be easily chained. For example, foo()?.bar() or foo(bar()?). Because Err(x)? can’t be used that way (it will always return), it is more clear to write return Err(x).

Предлагаете заглушить?

Reply:
@alexlapa hmm.. I didn't have that complain 😕  
But linter advise with `.into()` is better 👍

Reply:
@tyranron ,

> hmm.. I didn't have that linter complain

Я на nightly сижу. Пожалуй, стоит так сделать c8ef56a39d3838033a51b8e784a3efff42fd0a49

Reply:
@alexlapa that explains stuff, yes. However, consider to use `stable`. `nightly` is too unstable to be used by default.